### PR TITLE
Adding Nicholas Decheine to fellows webpage

### PR DESCRIPTION
**Pull request by decheine**

### CODE REVIEW

File: pages/fellows/decheine.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+photo: /assets/images/team/Nicholas-Decheine.jpg
+institution: University of Wisconsin - Madison
+e-mail: decheine@hep.wisc.edu
+project_title: Implementing ServiceX Data as a Source for ROOT’s RDataFrame
+project_goal: >
+    ServiceX is a smart data delivery service used by physicists to retrieve data subsets for analysis. This project aims to develop a streamlined C++ software service that bridges the gap between ServiceX experiment data acquisition and the creation and instantiation of a ROOT RDataFrame for analysis. It will accomplish this by fetching ServiceX data using a user request, construct an RDataFrame instance using the fetched data, and return it to the user, ready for analysis. This utility will streamline the analysis process so that physicist will spend less time on organizing their data and more time doing meaningful analysis on a virtual data frame. The demo dataset will be CMS Higgs event data from CERN’s Open Data collection.
+mentors:
+
+  - Gordon Watts - (University of Washington)
+
```
Comment (henryiii) [inline pages/fellows/decheine.md]:
```suggestion
mentors:
- gordonwatts
```

**Comment by henryiii** [comment]
Looks like we don't really advertise that short names work there, but they do.

**Comment by decheine** [comment]
Okay, is everything good with the pull request then?

**Comment by henryiii** [comment]
Yep, thanks!

**Pull request merged**